supplementary_style_guide/glossary_terms_conventions/general_conventions/p.adoc

```diff
@@ -64,6 +140,30 @@
 
 *See also*: xref:logical-topology[logical topology], xref:signal-topology[signal topology]
 
+// EAP: Added "In Red Hat JBoss Enterprise Application Platform,"
+[discrete]
+[[picketlink-federation]]
+==== image:images/yes.png[yes] picketlink-federation subsystem (noun)
+*Description*: In Red Hat JBoss Enterprise Application Platform, the "picketlink-federation" subsystem is used to configure single sign-on (SSO) using security assertion markup language (SAML). In general text, write in lowercase as two words separated by a hyphen. Use "PicketLink Federation subsystem" when referring to the picketlink-federation subsystem in titles and headings. When writing the term in its heading form, ensure that you include an uppercase 'L'.
```

Comment:
Should "security assertion markup language" be capitalized? According to the IBMSG, "Most spelled-out forms of language and protocol names have initial uppercase letters." Also, I do have the same question as with TOML about whether we should spell it out at all.